api/src/main/java/com/codeforcommunity/exceptions/IncorrectBlockStatusException.java

```diff
@@ -1,14 +1,17 @@
 package com.codeforcommunity.exceptions;
 
+import com.codeforcommunity.enums.ReservationAction;
 import com.codeforcommunity.rest.FailureHandler;
 import io.vertx.ext.web.RoutingContext;
 
+import java.util.List;
+
 public class IncorrectBlockStatusException extends HandledException {
 
   private final int blockId;
-  private final String expectedStatus;
+  private final List<ReservationAction> expectedStatus;
```

Comment:
Probably want to change this to "expectedStatuses" since we're taking a list now